api/src/components/publication/controller.ts

```diff
@@ -67,6 +67,28 @@ export const get = async (
     }
 };
 
+export const getPubSeed = async (
+    event: I.APIRequest<undefined, I.GetPublicationQueryParams>
+): Promise<I.JSONResponse> => {
+    try {
+
+        const publication = await publicationService.getPubSeed(event.queryStringParameters.title);
+       
+        if(publication !== null) {
```

Comment:
"publication" will never be null because "findMany" always returns an array with the records that match the "where" clause, or an empty array if there's no match. There's no need to check for null here